tests/device_tests/cardano/test_sign_tx.py

```diff
@@ -92,46 +95,31 @@ def test_cardano_sign_tx_failed(client, parameters, result):
             )
 
 
-@parametrize_using_common_fixtures("cardano/sign_tx.chunked.json")
-def test_cardano_sign_tx_with_multiple_chunks(client, parameters, result):
-    inputs = [cardano.parse_input(i) for i in parameters["inputs"]]
-    outputs = [cardano.parse_output(o) for o in parameters["outputs"]]
-    certificates = [cardano.parse_certificate(c) for c in parameters["certificates"]]
-    withdrawals = [cardano.parse_withdrawal(w) for w in parameters["withdrawals"]]
-    auxiliary_data = cardano.parse_auxiliary_data(parameters["auxiliary_data"])
+def _transform_expected_result(result):
```

Comment:
side note: this (and the whole `trezorlib.cardano` changeset) is a strong argument for doing the trezorlib part _somehow differently_, but I honestly don't have any idea how. Maybe follow how Connect does it?

No change requested (I don't think anyone other than the test suite is actually using the Python API), but noting it for posterity.